java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -1231,11 +1237,11 @@ class DatabaseImpl(
         arrayOf(collectionId.toString())
     ).map {
         Reference(
-            id = it.getString(0),
-            storageKey = StorageKeyParser.parse(it.getString(3)),
-            version = it.getVersionMap(4),
-            creationTimestamp = it.getString(1).toLong(),
-            expirationTimestamp = it.getString(2).toLong()
+            it.getString(0),
+            StorageKeyParser.parse(it.getString(3)),
+            it.getVersionMap(4),
+            it.getString(1).toLong(),
+            it.getString(2).toLong()
```

Comment:
I found the named arguments helpful here. You don't really know what they all mean without them; 5 args is a little hard to remember off the top of your head.